pytest.ini

```diff
@@ -13,6 +13,7 @@ filterwarnings =
     ignore::UserWarning:cpuinfo,
 # Matrix PendingDeprecationWarning.
     ignore:the matrix subclass is not
+    ignore:Importing from numpy.matlib is
```

Comment:
I think we want to leave this where it was - we should not be emitting theses warnings anywhere else.

Reply:
Ah, nevermind